src/Entity/PropertyMapper.php

```diff
@@ -104,8 +104,10 @@ protected function setConfig() {
   public static function getConfig($type, $bundle) {
     $paths = module_invoke_all('aws_sqs_entity_property_mapper_config_paths');
     $file_pattern = join('.', [$type, $bundle, 'yml']);
+    $file_pattern_alt = join('.', [$type, 'yml']);
```

Comment:
@mauricio-molina love it. Do you mind changing the vars to `*_type` instead of `*_alt`? I suggest only so the name carries that meaning.

BTW, I initially wondered if it could be useful to invoke a Drupal hook to define the file name pattern (as we do just above for config paths), but I just never did that, preferring instead to keep it simple for the meantime. I still like that idea here (keeping it simple however possible), but just wanted to make my thought process on this transparent here since you're adding some new functionality. It's a conversation 😄